### PR TITLE
Completed exact change, record collection, and symmetric difference a…

### PR DESCRIPTION
**Pull request by eobaah**
…lgorithms

### CODE REVIEW

File: algorithms/exact_change.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/*
+https://www.freecodecamp.com/challenges/exact-change
```
Comment (rae-ralston) [inline algorithms/exact_change.js:2]: take out all comments for code review

File: algorithms/exact_change.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  { name: 'PENNY', val: 0.01}
+];
+
+
+function checkCashRegister(price, cash, cid) {
+  var change = cash - price;
+
+
+  // Transform CID array into drawer object
+  var register = cid.reduce(function(accumulator, current) {
```
Comment (deonna) [inline algorithms/exact_change.js:38]: Nice use of `reduce()`. I'd decompose this and create a separate function for transforming the CID array (Single Responsibility Principle).

File: algorithms/exact_change.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    return 'Closed';
+  }
+
+  // Handle obvious insufficent funds
+  if (register.total < change) {
+    return 'Insufficient Funds';
+  }
+
+  // Loop through the denomination array
+  var change_arr = denom.reduce(function(accumulator, current) {
```
Comment (deonna) [inline algorithms/exact_change.js:55]: I'd call the accumulator `change` or `changeAccumulator` to be more specific to the data you're dealing with. I'd also decompose this into a separate function called `getChange()` or something similar.

Also, be careful of maintaining code style consistency: you've camel cased earlier functions and have used `_` in others. Pick one (JS convention is camel case) and make sure you stick to it. :)

File: algorithms/exact_change.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+ due. Return the string "Closed" if cash-in-drawer is equal to the change due.
+
+Otherwise, return change in coin and bills, sorted in highest to lowest order.
+
+
+*/
+
+
+// Create an object which hold the denominations and their values
+var denom = [
```
Comment (deonna) [inline algorithms/exact_change.js:20]: More explicit naming could be helpful, e.g., `nameToDenomination`.

Also, why not have an object that maps the string to the value itself, e.g., 

```javascript

const nameToDenomination = {
  'ONE HUNDRED': 100.00,
  ...
}
```

**Review by deonna — APPROVED** [review]
Good job - like the comments and pseudocode you included. Very methodical. :)

**Pull request merged**